components/script/dom/path2d.rs

```diff
@@ -175,3 +260,293 @@ impl Path2DMethods<crate::DomTypeHolder> for Path2D {
         )
     }
 }
+
+#[cfg(test)]
+mod test {
```

Comment:
I think we can remove this tests as we are relying on kurbo's implementation here. We could also upstream those tests to kurbo.